src/VisualStudio/IntegrationTest/TestUtilities/RetryHelper.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities
+{
+    public static class RetryHelper
+    {
+        public static void Try(
+           Action action,
+           int retryCount = 3)
+        {
+            Try<object>(() =>
+            {
+                action();
+                return null;
+            }, retryCount);
+        }
+
+        public static T Try<T>(
+            Func<T> action,
+            int retryCount = 3)
```

Comment:
Yes. The original design decision, after much discussion with @jasonmalinowski, @DustinCampbell, and @Pilchie was that we should just retry indefinitely for things which aren't explicitly expected to pass or be available the first time.